Towny/src/main/java/com/palmergames/bukkit/towny/db/SQLSchema.java

```diff
@@ -72,10 +75,11 @@ private static void initTable(Connection cntx, TownyDBTableType tableType) {
 	 */
 	private static String fetchTableSchema(TownyDBTableType tableType) {
 		return switch(tableType) {
-		case TOWNBLOCK -> fetchCreateTownBlocksStatement();
-		case JAIL -> fetchCreateUUIDStatement(tableType);
-		case PLOTGROUP -> fetchCreatePlotGroupStatement(tableType);
-		default -> fetchCreateNamedStatement(tableType);
+			case TOWNBLOCK -> fetchCreateTownBlocksStatement();
+			case JAIL -> fetchCreateUUIDStatement(tableType);
+			case PLOTGROUP -> fetchCreatePlotGroupStatement(tableType);
+			case COOLDOWN -> "CREATE TABLE IF NOT EXISTS " + TABLE_PREFIX + tableType.tableName() + " (`key` varchar(200) not null, primary key (`key`))";

```

Comment:
please extract this to a method so that it is in the same style as the other switches.